### PR TITLE
2019 updates

### PR DESCRIPTION
**Pull request by RobinsonZ**
Don't merge at the moment (particularly since @edelmanjm kind of needs to re-add the test framework), but should be ready. May need some final dependency version updates, I stole the vender dependency JSONs off of 5190's Github and they may or may not be current. Also the documentation about how to pull this down needs to be revised to include aforementioned vendor JSONs.

The testing framework is broken because CTRE made some changes where Victors (and thus `IMotorController` implementors) no longer have the ability to do things they actually can't do such as get the current and some sensor position stuff.

### CODE REVIEW

**Comment by edelmanjm** [comment]
@RobinsonZ Send help with how to fix issue :(

**Comment by edelmanjm** [comment]
Closes #99

File: gradlew [hunk 1]
```diff
@@ -28,7 +28,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m"'
```
Comment (edelmanjm) [inline gradlew]: You sure?

File: src/main/java/org/team1540/rooster/wrappers/ChickenVictor.java [hunk 2]
```diff
@@ -16,7 +16,7 @@
 import com.ctre.phoenix.motorcontrol.can.MotControllerJNI;
 import com.ctre.phoenix.motorcontrol.can.VictorSPX;
 
-public class ChickenVictor extends VictorSPX implements ChickenController {
```
Comment (edelmanjm) [inline src/main/java/org/team1540/rooster/wrappers/ChickenVictor.java]: ...no. We need to update the ChickenController interface, not the other way around.

File: gradlew (hunk 1 above)
Comment (RobinsonZ) [reply gradlew]: That's an automated change by Gradle.

**Pull request merged**